x/service/module/autocli.go

```diff
@@ -0,0 +1,51 @@
+package service
+
+import (
+	autocliv1 "cosmossdk.io/api/cosmos/autocli/v1"
+)
+
+// AutoCLIOptions implements the autocli.HasAutoCLIConfig interface.
+func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
+	return &autocliv1.ModuleOptions{
+		// TODO_TECHDEBT(#370): Integrate with AutoCLI
```

Comment:
Uncomment the `autocliv1.ModuleOptions` fields and comment the `[]autocliv1.RpcCommandOptions` elements instead.

Reply:
I agree that it is "safe" in the sense that it doesn't seem to effect runtime behavior; however, I think it could lead to confusion, especially for onboarding developers.